packages/datadog-plugin-openai/src/index.js

```diff
@@ -165,11 +169,14 @@ class OpenApiPlugin extends TracingPlugin {
     const fullStore = storage.getStore()
     const store = fullStore.openai
 
+    try {
+      path = new URL(path).pathname
+    } catch { /* path is already a URL pathname */ }
```

Comment:
As with the object case above, can we just do a check rather than a try/catch? Exceptions are expensive.